case/src/test/kotlin/com/ritense/case/web/dto/CaseSettingsDtoTest.kt

```diff
@@ -30,4 +30,23 @@ class CaseSettingsDtoTest {
         assertNull(caseSettingsDto.canHaveAssignee)
         assertFalse(updatedCaseSettings.canHaveAssignee)
     }
+
+    @Test
+    fun `should set autoAssignTasks to false when canHaveAssignee is set to false`() {
+        val currentCaseSettings = CaseDefinitionSettings(
+            name = "case-name",
+            canHaveAssignee = true,
+            autoAssignTasks = true
+        )
+        val caseSettingsDto = CaseSettingsDto(
+            canHaveAssignee = false
+        )
+        val updatedCaseSettings = caseSettingsDto.update(currentCaseSettings)
+
+        assertTrue(currentCaseSettings.canHaveAssignee)
+        assertTrue(currentCaseSettings.autoAssignTasks)
+        assertNull(caseSettingsDto.autoAssignTasks)
+        assertFalse(updatedCaseSettings.canHaveAssignee)
+        assertFalse(updatedCaseSettings.canHaveAssignee)
```

Comment:
You're asserting the same thing twice here, this should probably be autoAssignTasks.

```suggestion
        assertFalse(updatedCaseSettings.autoAssignTasks)
```